lib/manager/components/ProjectSettings.js

```diff
@@ -1,22 +1,27 @@
 // @flow
 
-import React, {Component} from 'react'
-import {Row, Col, Panel, ListGroup, ListGroupItem} from 'react-bootstrap'
-import {LinkContainer} from 'react-router-bootstrap'
+import React, { Component } from 'react'
+import { Row, Col, Panel, ListGroup, ListGroupItem } from 'react-bootstrap'
+import { LinkContainer } from 'react-router-bootstrap'
+
+import { deleteProject, updateProject } from '../actions/projects'
```

Comment:
It's not done here, but we generally import actions into components using the `import * as projectActions from '../actions/projects'` syntax and reference actions for flow types like:

```
updateProject: typeof projectActions.updateProject
```

Reply:
Interesting! I prefer the style you've mentioned. Refactored!